assets/js/nav.js

```diff
@@ -0,0 +1,32 @@
+const MOBILE_WIDTH_BREAKPOINT = 770;
```

Comment:
How'd you decide on using `770`?

Reply:
If you look at the header SCSS file, 770 is what is used as the breakpoint for mobile styling so I went with that.